nominees/covid-19-sms.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Covid-19-SMS",
+  "description": "A nodejs api to send sms msg using africatalking api for delevering information about COVID-19 virus.",
+  "website": "https://github.com/africa-covid-19-response-toolkit/awareness-sms",
```

Comment:
@rachita19082 kindly substitute the current Github repo for an actual website URL for the website field. Thanks.